### ISSUE TITLE
`rake bullet_train:themes:light:eject` should eject partials from _all_ parent themes.

### ISSUE THREAD

**Issue by andrewculver** [bullet-train-co/bullet_train-themes-light#59]

**Comment by gazayas** [comment]
Yes, it looks like only the light theme partials are being ejected. I'll look into `base` and `tailwind_css` to see which way is best to eject the rest of the partials.

![image](https://user-images.githubusercontent.com/10546292/199914147-59e4d19f-c821-4055-b0dc-ce33c0314b8c.png)
